syntax = "proto3";

message OutDir {
  string dir = 3;
}

message EdfDir {
  string dir = 1;
}

service AuraDataprep {
  rpc prepareEdfDir(EdfDir) returns(OutDir);
}
